src/Uno.UWP/Globalization/NumberFormatting/CurrencyFormatter.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
```

Comment:
@jeromelaban Hi, I'm using "#nullable" wherever possible but we can't use it here because when CurrencyFormatter (and other formatters) initializes in UWP app default value for "NumberRounder" property is null. If we want to enable nullable we should change the "INumberRounderOption" Interface from `INumberRounder NumberRounder { get; set; }` to `INumberRounder? NumberRounder { get; set; }`

Reply:
Let's do this then, it makes sense to add nullability there. Thanks!